local-dev/git/entrypoint.sh

```diff
@@ -6,4 +6,4 @@ fi
 
 ls -d -- git/*
 
-exec /sbin/runsvdir /etc/service
+exec /usr/sbin/runsvdir /etc/service
```

Comment:
@tobybellwood I'm a bit unsure why this shows as being diff, when its already merged in https://github.com/uselagoon/lagoon/pull/3869 ?

Reply:
Ah, cherry picked.

Reply:
@CGoodwin90 can you merge main and push again please?